ts/sveltelib/input-manager.ts

```diff
@@ -71,6 +71,17 @@ function getInputManager(): InputManager {
         cancelInsertText();
     }
 
+    function onInput(this: HTMLElement): void {
```

Comment:
I'm somewhat opposed to using `this` like this, as I find it to be a bit magical.
First it doesn't behave the same in arrow functions, and second it's not really clear which target it is actually binding to (`currentTarget` vs `target`).

Reply:
Thanks @hgiesel,
I've changed to use `event.currentTarget` instead of `this`.